### PR TITLE
Add sounds and rework the existing textures to custom Dwarven Mines /… 

### PR DESCRIPTION
**Pull request by romangraef**

### CODE REVIEW

**Comment by romangraef** [comment]
That pr was meant to go into :dokm, but i think this also could go into :master

File: src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Mining.java [hunk 1]
```diff
@@ -499,4 +499,28 @@
     )
     @ConfigEditorBoolean
     public boolean dwarvenTextures = false;
+    @Expose
+    @ConfigOption(
+            name = "Crystal Hollows Textures",
+            desc = "Allows texture packs to retexture blocks in the Crystal Hollows. If you don't have a texturepack that does this, you should leave this off"
```
Comment (NopoTheGamer) [inline src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Mining.java:505]:
```suggestion
            desc = "Allows texture packs to retexture blocks in the Crystal Hollows. If you don't have a texture pack that does this, you should leave this off"
```
you put a space between the other texture packs

File: src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java [hunk 2]
```diff
@@ -95,8 +99,23 @@ public Commands() {
         ClientCommandHandler.instance.registerCommand(neuHelp);
         ClientCommandHandler.instance.registerCommand(neuFeatures);
         ClientCommandHandler.instance.registerCommand(neuRepoMode);
+        ClientCommandHandler.instance.registerCommand(dokmTest);
     }
 
+    SimpleCommand dokmTest = new SimpleCommand("dokmTest", new SimpleCommand.ProcessCommandRunnable() {
```
Comment (NopoTheGamer) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java:105]: move this to /neudevtest

**Review by NopoTheGamer — CHANGES_REQUESTED** [review]

**Review by DoKM — APPROVED** [review]

**Comment by NopoTheGamer** [comment]
it seems to work fine but it would be nice if the audio didn't [stack](https://notenoughupdates.discord.com/channels/516977525906341928/832652653292027904/930989575944413275)

**Review by NopoTheGamer — APPROVED** [review]

**Review by NopoTheGamer — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

File: src/main/resources/assets/notenoughupdates/sounds/titaniumbreak.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"debouncer":20}
```
Comment (IRONM00N) [inline src/main/resources/assets/notenoughupdates/sounds/titaniumbreak.json:1]:
```suggestion
{ "debouncer": 20 }
```

File: src/main/resources/assets/notenoughupdates/sounds/mithrilbreak.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"debouncer":20}
```
Comment (IRONM00N) [inline src/main/resources/assets/notenoughupdates/sounds/mithrilbreak.json:1]:
```suggestion
{ "debouncer": 20 }
```

File: src/main/resources/assets/notenoughupdates/sounds/gemstonetopazbreak.json [hunk 5]
```diff
@@ -0,0 +1 @@
+{"debouncer":20}
```
Comment (IRONM00N) [inline src/main/resources/assets/notenoughupdates/sounds/gemstonetopazbreak.json:1]:
```suggestion
{ "debouncer": 20 }
```

File: src/main/resources/assets/notenoughupdates/sounds/gemstonesapphirebreak.json [hunk 6]
```diff
@@ -0,0 +1 @@
+{"debouncer":20}
```
Comment (IRONM00N) [inline src/main/resources/assets/notenoughupdates/sounds/gemstonesapphirebreak.json:1]:
```suggestion
{ "debouncer": 20 }
```

File: src/main/resources/assets/notenoughupdates/sounds/gemstonerubybreak.json [hunk 7]
```diff
@@ -0,0 +1 @@
+{"debouncer":20}
```
Comment (IRONM00N) [inline src/main/resources/assets/notenoughupdates/sounds/gemstonerubybreak.json:1]:
```suggestion
{ "debouncer": 20 }
```

File: src/main/resources/assets/notenoughupdates/sounds/gemstonejasperbreak.json [hunk 8]
```diff
@@ -0,0 +1 @@
+{"debouncer":20}
```
Comment (IRONM00N) [inline src/main/resources/assets/notenoughupdates/sounds/gemstonejasperbreak.json:1]:
```suggestion
{ "debouncer": 20 }
```

File: src/main/resources/assets/notenoughupdates/sounds/gemstonejadebreak.json [hunk 9]
```diff
@@ -0,0 +1 @@
+{"debouncer":20}
```
Comment (IRONM00N) [inline src/main/resources/assets/notenoughupdates/sounds/gemstonejadebreak.json:1]:
```suggestion
{ "debouncer": 20 }
```

File: src/main/resources/assets/notenoughupdates/sounds/gemstoneamethystbreak.json [hunk 10]
```diff
@@ -0,0 +1 @@
+{"debouncer":20}
```
Comment (IRONM00N) [inline src/main/resources/assets/notenoughupdates/sounds/gemstoneamethystbreak.json:1]:
```suggestion
{ "debouncer": 20 }
```

File: src/main/resources/assets/notenoughupdates/sounds/gemstoneamberbreak.json [hunk 11]
```diff
@@ -0,0 +1 @@
+{"debouncer":20}
```
Comment (IRONM00N) [inline src/main/resources/assets/notenoughupdates/sounds/gemstoneamberbreak.json:1]:
```suggestion
{ "debouncer": 20 }
```

File: src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java [hunk 12]
```diff
@@ -952,7 +971,8 @@ public void processCommand(ICommandSender sender, String[] args) {
 
                         NotEnoughUpdates.INSTANCE.colourMap[x][y] = new Color(entry.getValue().getAsInt(), true);
                     }
-                } catch (Exception ignored) {}
+                } catch (Exception ignored) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java:974]:
```suggestion
                } catch (Exception ignored) {}
```

File: src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java [hunk 13]
```diff
@@ -952,7 +971,8 @@ public void processCommand(ICommandSender sender, String[] args) {
...
+                }
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java:975]:
```suggestion
```

File: src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java [hunk 14]
```diff
@@ -916,7 +934,8 @@ public void processCommand(ICommandSender sender, String[] args) {
                     Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText(""));
                     NotEnoughUpdates.INSTANCE.displayLinks(update);
                     Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText(""));
-                } catch (Exception ignored) {}
+                } catch (Exception ignored) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java:937]:
```suggestion
                } catch (Exception ignored) {}
```

File: src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java [hunk 15]
```diff
@@ -916,7 +934,8 @@ public void processCommand(ICommandSender sender, String[] args) {
...
+                }
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java:938]:
```suggestion
```

File: src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java [hunk 16]
```diff
@@ -372,7 +378,8 @@ public void processCommand(ICommandSender sender, String[] args) {
             if (NotEnoughUpdates.INSTANCE.getConfigFile().exists()) {
                 try (BufferedReader reader = new BufferedReader(new InputStreamReader(new FileInputStream(NotEnoughUpdates.INSTANCE.getConfigFile()), StandardCharsets.UTF_8))) {
                     NotEnoughUpdates.INSTANCE.config = NotEnoughUpdates.INSTANCE.manager.gson.fromJson(reader, NEUConfig.class);
-                } catch (Exception ignored) {}
+                } catch (Exception ignored) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java:381]:
```suggestion
                } catch (Exception ignored) {}
```

File: src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java [hunk 17]
```diff
@@ -372,7 +378,8 @@ public void processCommand(ICommandSender sender, String[] args) {
...
+                }
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java:382]:
```suggestion
```

**Review by IRONM00N — CHANGES_REQUESTED** [review]

**Comment by romangraef** [comment]
you serious?

**Comment by IRONM00N** [comment]
> you serious?

:)

**Comment by jani270** [comment]
> you serious?

yes he is

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**